lib/CMakeLists.txt

```diff
@@ -9,7 +9,9 @@ set (QUDA_OBJS
   eigensolve_quda.cpp quda_arpack_interface.cpp
   multigrid.cpp transfer.cpp block_orthogonalize.cu inv_bicgstab_quda.cpp
   prolongator.cu restrictor.cu staggered_prolong_restrict.cu
-  gauge_phase.cu timer.cpp malloc.cpp
+  gauge_phase.cu timer.cpp
+  targets/${QUDA_TARGET_DIR}/malloc.cpp
```

Comment:
should also be updated with `target_sources`

Reply:
Is the `PUBLIC` keyword correct here: https://github.com/lattice/quda/commit/087daceb0cd9a69db9a7660d7484d36167869e21